package.json

```diff
@@ -23,12 +23,15 @@
     "node": ">=8.0.0",
     "yarn": "^1.0.0"
   },
+  "resolutions": {
+    "@types/react": "^17.0.40"
+  },
```

Comment:
Why do we need this? What libraries do require this override?

Reply:
https://stackoverflow.com/a/73005024

Reply:
yes yes, I understand what it does. the problem is, that basically now we override all the dependencies to use this specific react types. proper solution would be to identify which dependency doesn't use `@types/react@17` and possibly update that dependency

Reply:
Aight, lemme check

Reply:
I removed it, and it still works, maybe something with `yarn.lock`